pkg/v1/providers/ytt/02_addons/management-packages/add_management_package_repo.yaml

```diff
@@ -0,0 +1,49 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+#@ load("/lib/helpers.star", "get_default_tkg_bom_data", "get_image_repo_for_component")
+
+#@ def management_package_repo():
+#@ management_package_repo_bundle = get_default_tkg_bom_data().components["tkg-management-packages"][0].images.tanzuManagementPackageRepositoryImage
```

Comment:
@rajathagasthya can you confirm if `tkg-management-packages` would be BOM entry for management packages?

Reply:
BOM entry is `tanzu-framework-management-packages` and image name is `tanzuFrameworkManagementPackageRepositoryImage`.